jekyll/_cci2/skip-build.md

```diff
@@ -117,22 +86,15 @@ Projects for which auto-cancel is enabled in the Advanced Settings will have wor
 
 **Notes:** It is important to carefully consider the impact of enabling the auto-cancel feature, for example, if you have configured automated deployment jobs on non-default branches. Auto-cancelling workflows requires enabling the preview [build processing]({{ site.baseurl }}/2.0/build-processing/) feature.
 
-1. In the CircleCI application,
-go to your project's settings
-by clicking the gear icon next to your project.
+1. In the CircleCI application, go to your project's settings by clicking the gear icon next to your project.
 
-2. In the **Build Settings** section,
-click on **Advanced Settings**.
+2. In the **Build Settings** section, click on **Advanced Settings**.
 
 3. In the **Enabling Build Processing (preview)** section, click the **On** button.
```

Comment:
@thomasr8 ^ this line needs to be changed.

Reply:
Fixed